app/views/posts/show.html.erb

```diff
@@ -1,2 +1,28 @@
-<h1>Post</h1>
-<!-- Placeholder for displaying a single post -->
+<main>
+  <div class="single-post-div">
+    <article class="post-article">
+      <div class="post-info">
+        <div>
+          <h3 class="post-title"><%= @post.title %></h3>
+          <p class="user-name">by <%= @user.name%></p>
+        </div>
+        <p class="posts-counter">Comment: <%= @post.comments_counter || 0 %></p>
+      </div>
+      <div class="post-details">
+        <p><%= @post.text %></p>
+      </div>
+    </article>
+
+    <ul class="comments-container">
+      <% if @comments.any? %>
+        <% @comments.each do |comment| %>
+          <li>
+            <p><%= comment.author.name %>: <%= comment.text %></p>
+          </li>
+        <% end %>
+      <% else %>
+        <li>No comments yet...</li>
+      <% end %>
+    </ul>
+  </div>
+</main>
```

Comment:
| @WilhelmK109 Post Show Page | Microverse Post Show Page |
|-|-|
| ![image](https://github.com/WilhelmK109/BlogApp/assets/93039359/e97e01ea-e171-4bd8-8361-76f9089db2ee) | ![image](https://github.com/WilhelmK109/BlogApp/assets/93039359/1d9e7f06-129a-4a69-95a6-4c82d0421b2c) |

- [ ] It would be nice if you add the likes counter into your design to make your app look identical to the given design which is more professional 👍
- [ ] Kindly make your Comments Counter and Likes Counter of the right side of the page to follow the given design to make your app look catchy 😊

Reply:
@HAMMAS-SALEEM, thanks so much for your comments and suggestions. I will make the necessary changes and re-submit them for review.